subgraph/src/fetch/account.ts

```diff
@@ -3,11 +3,20 @@ import { Address } from "@graphprotocol/graph-ts";
 import { Account } from "../../generated/schema";
 
 export function fetchAccount(address: Address): Account {
-  let account = Account.load(address.toHex());
+  let account = Account.load(addressToHex(address));
 
   if(!account) {
-    account = new Account(address.toHex())
+    account = new Account(addressToHex(address))
   }
   account.save();
   return account;
 }
+
+export function addressToHex(id: Address): string {
```

Comment:
would web3's padLeft help here?
https://web3js.readthedocs.io/en/v1.2.0/web3-utils.html#padleft

Reply:
Maybe? I haven't used web3js and figured it was small enough to not import a libary.